mscore/articulationprop.cpp

```diff
@@ -58,7 +58,7 @@ ArticulationProperties::ArticulationProperties(Articulation* na, QWidget* parent
 
             for (const Channel* a : instrument->channel()) {
                   if (a->name().isEmpty() || a->name() == Channel::DEFAULT_NAME) {
-                        channelList->addItem(tr(Channel::DEFAULT_NAME));
```

Comment:
this never worked apparently, that string ('normal', prior to this PR) never made it to Transifex